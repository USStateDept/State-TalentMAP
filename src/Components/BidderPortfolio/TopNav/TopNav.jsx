import { BIDDER_PORTFOLIO_COUNTS } from '../../../Constants/PropTypes';
import Navigation from './Navigation';

const TopNav = ({ bidderPortfolioCounts }) => (
  <div className="usa-grid-full portfolio-top-nav-container">
    <div className="usa-width-one-whole">
      <Navigation counts={bidderPortfolioCounts} />
    </div>
  </div>
);

TopNav.propTypes = {
  bidderPortfolioCounts: BIDDER_PORTFOLIO_COUNTS.isRequired,
};

export default TopNav;
