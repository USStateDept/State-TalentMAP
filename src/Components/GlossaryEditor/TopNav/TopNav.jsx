import NavigationList from './Navigation/NavigationList';

const TopNav = () => (
  <div className="usa-grid-full portfolio-top-nav-container">
    <div className="usa-width-five-sixths">
      <NavigationList />
    </div>
  </div>
);

export default TopNav;
