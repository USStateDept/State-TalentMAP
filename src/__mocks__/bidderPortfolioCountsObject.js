const bidderPortfolioCountsObject = {
  all_clients: 100,
  bidding_clients: 50,
  in_panel_clients: 20,
  on_post_clients: 30,
  bidding_no_handshake: 10,
};

export default bidderPortfolioCountsObject;
