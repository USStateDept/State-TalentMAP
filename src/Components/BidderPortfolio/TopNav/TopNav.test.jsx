import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import TopNav from './TopNav';
import bidderPortfolioCountsObject from '../../../__mocks__/bidderPortfolioCountsObject';

describe('TopNavComponent', () => {
  it('is defined', () => {
    const wrapper = shallow(
      <TopNav
        onUpdate={() => {}}
        bidderPortfolioCounts={bidderPortfolioCountsObject}
      />,
    );
    expect(wrapper).toBeDefined();
  });

  it('matches snapshot', () => {
    const wrapper = shallow(
      <TopNav
        onUpdate={() => {}}
        bidderPortfolioCounts={bidderPortfolioCountsObject}
      />,
    );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
