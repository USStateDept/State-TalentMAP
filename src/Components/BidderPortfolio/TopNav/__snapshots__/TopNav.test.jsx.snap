// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopNavComponent matches snapshot 1`] = `
<div
  className="usa-grid-full portfolio-top-nav-container"
>
  <div
    className="usa-width-one-whole"
  >
    <Navigation
      counts={
        Object {
          "all_clients": 100,
          "bidding_clients": 50,
          "bidding_no_handshake": 10,
          "in_panel_clients": 20,
          "on_post_clients": 30,
        }
      }
    />
  </div>
</div>
`;
